packages/cellxgene/meta.yaml

```diff
@@ -0,0 +1,29 @@
+name: cellxgene
+description: |
+    CZ CELLxGENE Annotate (pronounced "cell-by-gene") is an interactive data explorer for single-cell datasets, 
+    such as those coming from the Human Cell Atlas.
+project_home: https://github.com/chanzuckerberg/cellxgene
+documentation_home: https://docs.cellxgene.cziscience.com/
+publications:
+    - 10.1101/2021.04.05.438318
+install:
+    pypi: cellxgene
+tags:
+    - HCA
+license: MIT
+version: 1.1.1
+authors:
+    - csweaver
+    - bkmartinjr
+    - seve
+    - maniarathi
+    - atolopko-czi
+    - mckinsel
+    - MDunitz
+    - colinmegill
+    - blrnw3
+    - freeman-lab
+    - bmccandless
+    - mweiden
+test_command: |
```

Comment:
@ivirshup, how should we deal with tests like this one: 
https://github.com/chanzuckerberg/cellxgene/blob/main/.github/workflows/push_tests.yml

or also cell rank: 
https://github.com/theislab/cellrank/blob/master/.github/workflows/test.yml

It's not always as easy as `pip install .[test] && pytest`. 

I still think `test_command` is a good field, though. Like this every box from our requirements checklist has an entry in the yaml file, which makes it very easy to review.